Changelog.md

```diff
@@ -1,5 +1,9 @@
 ### 0.8.14 (unreleased)
 
+Important Bugfixes:
+ * Type Checker: Only allow changing data location between ``memory`` and ``calldata`` when overriding external functions.
```

Comment:
I think it would be clearer to state what is getting disallowed:
```suggestion
 * Type Checker: Disallow changing data location between ``memory`` and ``calldata`` when overriding internal functions.
```

Reply:
if we write 'internal', it could mean that it does not refer to public functions.